api/src/main/java/org/apache/iceberg/variants/VariantArray.java

```diff
@@ -20,6 +20,25 @@
 
 /** An variant array value. */
 public interface VariantArray extends VariantValue {
+  static String asString(VariantArray arr) {
```

Comment:
I'd prefer to have the `toString` implementation at the end of the file like the other interfaces. When people read this file, it is most likely that they are interested in the methods of the interface, not the `toString` helper.